### ISSUE TITLE
When running multiple app instances, only one provides "Calls" feature

### ISSUE THREAD

**Issue by bubble75** [IsmaelMartinez/teams-for-linux#275]
I need to participate in several teams simultaneously. That could be achieved by using native-packaged application (rpm in my case) by overriding XDG_CONFIG_HOME env variable, so teams-for-linux uses different config directories for different instances.

Unfortunately, only first-spawn instance gets access to sound equipment (it shows both "Calls" tab and "Calls" menu in Settings.
All other copies do not show any of them.

Also, once instance missed its sound, you need to sign-out and sign-in again in order to get sound back (assuming no other instances run at that time) - just restarting application does not help.

It would be really nice to get sound in all instances.

System is Fedora30 with pulseaudio.
 --- Solution:
Adding this to the code should be as easy as adding:

app.commandLine.appendSwitch('--try-supported-channel-layouts');
around https://github.com/IsmaelMartinez/teams-for-linux/blob/develop/app/index.js#L10

maybe someone can create a PR with that change.

**Comment by IsmaelMartinez** [comment]
maybe this is related/solves the issue? https://discuss.atom.io/t/enable-multiple-channel-audio/52945

basically, if you pass that as the argument and works, it will be fairly easy to integrate in the app.

**Comment by bubble75** [comment]
Heh, that helps!
Will test more thoroughly tomorrow and return.

**Comment by bubble75** [comment]
Found a way to test it very quickly.
Yes, that's it.

**Comment by IsmaelMartinez** [comment]
Adding this to the code should be as easy as adding:

```javascript
app.commandLine.appendSwitch('--try-supported-channel-layouts');
```
around https://github.com/IsmaelMartinez/teams-for-linux/blob/develop/app/index.js#L10

maybe someone can create a PR with that change.

**Comment by rdlittle** [comment]
@bubble75 
> Found a way to test it very quickly.
What method did you find in order to do your testing?

**Comment by bubble75** [comment]
I entered settings in both instances and called voicemail configuration. Both channels have been heard simultaneously, and pulseaudio shown all playback and capture streams.

**Comment by bubble75** [comment]
But I didn't try to record anything. And I had mic issues at the followup meeting, but haven't got a chance to try to resolve them, I just rejoined from android. Need to try once more.

**Comment by IsmaelMartinez** [comment]
let me how it goes, hopefully it can be an easy one for hacktoberfest if is the "fix" is the try-supported-channel-layouts...

**Comment by IsmaelMartinez** [comment]
tidying issues easy to fix to release 1.0.0 soon.
